src/card.css

```diff
@@ -0,0 +1,102 @@
+html {
+  --mainColor: #00acdc;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+}
+
+.card__product-image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto 40px;
+}
+
+.card__product-name {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  color: #060b35;
+  margin-bottom: 0;
+  margin-top: 40px;
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card___stars-box {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
+}
+
+.card__reviews {
+  text-align: right;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  padding-top: 24px;
+}
+
+.card__value-text {
+  text-align: left;
+  flex-grow: 1;
```

Comment:
You dont need it here and in price-number. Use flexbox on parent element to style them